parents/integrationtest/pom.xml

```diff
@@ -616,38 +787,113 @@
+            </properties>
+        </profile>
+
+        <!-- DB2 Docker container for tests -->
+        <!-- See test.database.run.db2.image.tag for the server version in use -->
+        <profile>
+            <id>ci-db2</id>
+            <properties>
+                <test.database.run.db2.skip>false</test.database.run.db2.skip>
+                <db.dialect>org.hibernate.dialect.DB297Dialect</db.dialect>
+                <jdbc.driver.groupId>com.ibm.db2</jdbc.driver.groupId>
+                <jdbc.driver.artifactId>jcc</jdbc.driver.artifactId>
+                <jdbc.driver.version>11.5.5.0</jdbc.driver.version>
+                <jdbc.driver>com.ibm.db2.jcc.DB2Driver</jdbc.driver>
+                <jdbc.url>jdbc:db2://localhost:50005/hreact</jdbc.url>
+                <jdbc.user>hreact</jdbc.user>
+                <jdbc.pass>hreact</jdbc.pass>
                 <jdbc.isolation />
             </properties>
         </profile>
 
+        <!-- Oracle Docker container for tests -->
+        <!-- See test.database.run.oracle.image.tag for the server version in use -->
+        <profile>
+            <id>ci-oracle</id>
+            <properties>
+                <test.database.run.oracle.skip>false</test.database.run.oracle.skip>
+                <db.dialect>org.hibernate.dialect.Oracle10gDialect</db.dialect>
+                <jdbc.driver.groupId>com.oracle.database.jdbc</jdbc.driver.groupId>
+                <jdbc.driver.artifactId>ojdbc8</jdbc.driver.artifactId>
+                <jdbc.driver.version>21.1.0.0</jdbc.driver.version>
+                <jdbc.driver>oracle.jdbc.OracleDriver</jdbc.driver>
+                <jdbc.url>jdbc:oracle:thin:@localhost:1521/XE</jdbc.url>
+                <jdbc.user>SYSTEM</jdbc.user>
+                <jdbc.pass>hibernate_orm_test</jdbc.pass>
+                <jdbc.isolation />
+            </properties>
+        </profile>
+
+        <!-- SQL Server Docker container for tests -->
+        <!-- See test.database.run.mssql.image.tag for the server version in use -->
+        <profile>
+            <id>ci-mssql</id>
+            <properties>
+                <test.database.run.mssql.skip>false</test.database.run.mssql.skip>
+                <db.dialect>org.hibernate.dialect.SQLServer2012Dialect</db.dialect>
+                <jdbc.driver.groupId>com.microsoft.sqlserver</jdbc.driver.groupId>
+                <jdbc.driver.artifactId>mssql-jdbc</jdbc.driver.artifactId>
+                <jdbc.driver.version>9.2.1.jre11</jdbc.driver.version>
+                <jdbc.driver>com.microsoft.sqlserver.jdbc.SQLServerDriver</jdbc.driver>
+                <jdbc.url>jdbc:sqlserver://localhost:1433;databaseName=tempdb</jdbc.url>
+                <jdbc.user>sa</jdbc.user>
+                <jdbc.pass>ActuallyRequired11Complexity</jdbc.pass>
+            </properties>
+        </profile>
+
         <!--
             ###################################################################
             Then, those with commercial drivers
```

Comment:
Can you clean this up, please? You obviously moved commercial drivers above this comment.

My suggestion:

* Remove this comment and every profile below this comment.
* Remove the comment higher up that contains "Profiles naming db instances in the Red Hat QA/QE lab".